src/engine/enginemaster.cpp

```diff
@@ -387,15 +387,17 @@ void EngineMaster::process(const int iBufferSize) {
 
     // Mix all the talkover enabled channels together.
     if (m_bRampingGain) {
-        ChannelMixer::mixChannelsRamping(
+        ChannelMixer::applyEffectsAndMixChannelsRamping(
```

Comment:
For a brief moment when ramping talkover channels on/off, EngineMaster::processChannel puts them in both m_activeTalkoverChannels and m_activeBusChannels, so I think they get processed twice in one callback. Might this cause issues? From brief testing it doesn't seem to cause the same feedback issue with the reverb effect that double processing the headphone channels did before. I suspect that is because they're only double processed very briefly while ramping.